WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -24,7 +24,7 @@ Todo:
     ```
     Linux: 
     ```
-     ~/bitcoin-[version number]/bin/bitcoin-qt -regtest -blockfilterindex -txindex -datadir=c:\Bitcoin
+     ~/bitcoin-[version number]/bin/bitcoin-qt -regtest -blockfilterindex -txindex -datadir=$HOME/.bitcoin/
```

Comment:
If it's `$HOME/.bitcoin`, then, unless something else is specified in `$HOME/.bitcoin/bitcoin.conf`, not need to specify `-datadir=` parameter at all.